### ISSUE TITLE
Do not recommend `sudo pip`

### ISSUE THREAD

**Issue by weskerfoot** [aws/aws-cli#4175]
https://github.com/aws/aws-cli#installation

Line 42 of the README recommends using `sudo pip install awscli`.

This is likely to completely break your Python installation! It is considered bad practice to mix system Python packages with ones installed via pip. This should not be recommended. `--user` should be recommended first. If you want to mention the possibility of installing with `sudo pip` then it should also be mentioned that it *will* cause breakage.

**Comment by JordonPhillips** [comment]
How it affects your system is highly dependent on the system and how you installed python. Even when it's a problem, the problem is that you're using pip at all.

For CLI v2 we're planning on distributing standalone installers for all platforms to avoid these and other issues.

**Comment by bengaywins** [comment]
When you say standalone installers, do we mean distro packaging via repos? Or just like the current self-contained tarball one can download for 1.x? Personally I much prefer the former rather than the latter as it now allows proper package management.

**Comment by morancj** [comment]
@bengaywins I strongly agree: I've opened [4842] for the consideration of @JordonPhillips and colleagues.
